src/plugins/notebook/components/Notebook.vue

```diff
@@ -1,16 +1,38 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2021, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+
 <template>
 <div class="c-notebook">
     <div class="c-notebook__head">
         <Search class="c-notebook__search"
                 :value="search"
-                @input="throttledSearchItem"
-                @clear="throttledSearchItem"
+                @input="search = $event"
```

Comment:
hmm interesting, I have never seen/used this pattern, what does it do?

Reply:
I'd be lying if I said I wasn't a little confused on this part. I modeled it after how I handled search in the tree. During the changes, I was running into issues with throttle (I'm just not familiar enough with it), so I switched out to just debounce any calls to the search results method. So here, whatever they type in is set to our search instance variable, then I set it up so we're just watching that for changes and calling the debounced getSearchResults method.

Reply:
lolz np, I am curious how it works. We both  can check this later 😄